src/components/shared/contextMenu/searchInput/searchInput.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+
+import { joinClasses } from 'utilities';
+
+import './searchInput.css';
+
+export const SearchInput = ({ children, className, ...rest }) => (
+  <input {...rest} type="text" className={joinClasses('context-menu-search-input', className)} />
```

Comment:
For consistent styling, should this use FormControl instead?

Reply:
No, we want different styling here and we don't want the extra dom elements the form control can render (like elements for errors, etcs).